src/main/commandServer/settingsValidator.ts

```diff
@@ -32,6 +32,8 @@ type SettingsValidationMapEntry<T> = {
   ValidatorFunc<T[k], T[k]> :
   T[k] extends Record<string, infer V> ?
   SettingsValidationMapEntry<T[k]> | ValidatorFunc<T[k], Record<string, V>> :
+  T[k] extends Array<infer V> ?
+  ValidatorFunc<T[k], Array<V>> :
```

Comment:
This ended up being unnecessary (because we ended up using a mapping instead of an array), but I guess it doesn't hurt to keep it?

Reply:
I guess this isn't the PR to add this in and as it's not used so I guess it's technically cruft.. removed. We can easily add this back if needed.